lib/jazzy/assets/js/jazzy.js

```diff
@@ -1,3 +1,13 @@
+window.jazzy = {'docset': false}
```

Comment:
In case someone's viewing this somewhere without access to the javascript files, maybe? But it's unlikely that would happen I think.